### PR TITLE
Rebranding pass plus refactor

### PR DESCRIPTION
**Pull request by OwenRichards1**
## Purpose
Remove references to Azure AD + 
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

* Test the code
```
Verify changes are correct
```

### CODE REVIEW

**Review by cilwerner — CHANGES_REQUESTED** [review]
Good start, just looking through and see that there's a lot of `Admin center`, or `Entra ID`, we need to be consistent with the rebranding -> `Microsoft Entra admin center`, and use https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/new-name for reference.

Won't do a full depth review for the moment until you've done another pass

File: web-app-aspnet/appsettings.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "AzureAd": {
     "Instance": "https://login.microsoftonline.com/",
-    "TenantId": "Enter the tenant ID obtained from the Azure portal",
-    "ClientId": "Enter the client ID obtained from the Azure portal",
+    "TenantId": "Enter_the_Application_Id_Here",
```
Comment (cilwerner) [inline web-app-aspnet/appsettings.json:4]: Wrong way round

File: console-cli/Program.cs [hunk 2]
```diff
@@ -4,10 +4,10 @@
 
 var config = new PublicClientApplicationOptions
 {
-    // 'Directory (tenant) ID' of app registration in the Azure portal - this value is a GUID
+    // 'Directory (tenant) ID' of app registration in the Admin center - this value is a GUID
```
Comment (cilwerner) [inline console-cli/Program.cs:7]:
```suggestion
    // 'Directory (tenant) ID' of app registration in the Microsoft Entra admin center - this value is a GUID

```
BULK edit

File: console-cli/Program.cs [hunk 3]
```diff
@@ -4,10 +4,10 @@
 
 var config = new PublicClientApplicationOptions
 {
-    // 'Directory (tenant) ID' of app registration in the Azure portal - this value is a GUID
+    // 'Directory (tenant) ID' of app registration in the Admin center - this value is a GUID
     TenantId = "",
 
-    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    // 'Application (client) ID' of app registration in Admin center - this value is a GUID
```
Comment (cilwerner) [inline console-cli/Program.cs:10]:
```suggestion
    // 'Application (client) ID' of app registration in Microsost Entra admin center - this value is a GUID

```

File: console-cli/Program.cs [hunk 4]
```diff
@@ -4,10 +4,10 @@
 
 var config = new PublicClientApplicationOptions
 {
-    // 'Directory (tenant) ID' of app registration in the Azure portal - this value is a GUID
+    // 'Directory (tenant) ID' of app registration in the Admin center - this value is a GUID
     TenantId = "",
```
Comment (cilwerner) [inline console-cli/Program.cs:8]:
```suggestion
    TenantId = "Enter_the_Tenant_ID_Here",

```
BULK edit

File: console-cli/Program.cs [hunk 5]
```diff
@@ -4,10 +4,10 @@
 
 var config = new PublicClientApplicationOptions
 {
-    // 'Directory (tenant) ID' of app registration in the Azure portal - this value is a GUID
+    // 'Directory (tenant) ID' of app registration in the Admin center - this value is a GUID
     TenantId = "",
 
-    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    // 'Application (client) ID' of app registration in Admin center - this value is a GUID
     ClientId = ""
```
Comment (cilwerner) [inline console-cli/Program.cs:11]:
```suggestion
    ClientId = "Enter_the_Application_ID_URI_Here"

```

File: console-daemon/Program.cs [hunk 6]
```diff
@@ -7,11 +7,11 @@
 var config = new {
     // Full directory URL, in the form of https://login.microsoftonline.com/<tenant_id>
     Authority = "",
```
Comment (cilwerner) [inline console-daemon/Program.cs:9]:
```suggestion
    Authority = "https://login.microsoftonline.com/Enter_the_Tenant_ID_Here",

```
BULK

File: console-daemon/Program.cs [hunk 7]
```diff
@@ -7,11 +7,11 @@
 var config = new {
     // Full directory URL, in the form of https://login.microsoftonline.com/<tenant_id>
     Authority = "",
-    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    // 'Application (client) ID' of app registration in Microsoft Admin center - this value is a GUID
     ClientId = "",
-    // Client secret 'Value' (not its ID) from 'Client secrets' in app registration in Azure portal
+    // Client secret 'Value' (not its ID) from 'Client secrets' in app registration in Microsoft Admin center
     ClientSecret = "",
```
Comment (cilwerner) [inline console-daemon/Program.cs:13]:
```suggestion
    ClientSecret = "Enter_the_Client_Secret_Value_Here",

```
BULK

File: console-daemon/Program.cs [hunk 8]
```diff
@@ -7,11 +7,11 @@
...
     Authority = "",
-    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    // 'Application (client) ID' of app registration in Microsoft Admin center - this value is a GUID
     ClientId = "",
-    // Client secret 'Value' (not its ID) from 'Client secrets' in app registration in Azure portal
+    // Client secret 'Value' (not its ID) from 'Client secrets' in app registration in Microsoft Admin center
     ClientSecret = "",
-    // Client 'Object ID' of app registration in Azure portal - this value is a GUID
+    // Client 'Object ID' of app registration in Microsoft Admin center - this value is a GUID
     ClientObjectId = ""
```
Comment (cilwerner) [inline console-daemon/Program.cs:15]:
```suggestion
    ClientObjectId = "Enter_the_Client_Object_ID_Here"

```

BULK

File: console-daemon/README.md
Comment (cilwerner) [inline console-daemon/README.md:1]: Microsoft Entra admin center spot on here, just add the Enter_XXX_Here for the strings.

File: desktop-winui/MainWindow.xaml.cs [hunk 9]
```diff
@@ -32,10 +32,10 @@ public MainWindow()
             msalPublicClientApp = PublicClientApplicationBuilder
                 .CreateWithApplicationOptions(new PublicClientApplicationOptions
                 {
-                     // 'Tenant ID' of your Azure AD instance - this value is a GUID
+                     // 'Tenant ID' of your Entra ID instance - this value is a GUID
```
Comment (cilwerner) [inline desktop-winui/MainWindow.xaml.cs:35]:
```suggestion
                     // 'Tenant ID' of your Microsoft Entra instance - this value is a GUID

```
BULK

File: desktop-winui/README.md [hunk 10]
```diff
@@ -43,7 +43,7 @@ Use these settings in your app registration.
 | **Platform type**               | **Mobile and desktop applications**                                 | Required value for this sample                                                  |
 | **Redirect URIs**               | `https://login.microsoftonline.com/common/oauth2/nativeclient`      | Required value for this sample                                                  |
 
-> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Microsoft Admin center, while `code formatting` indicates a value you enter into a text box in the Microsoft Admin center.
```
Comment (cilwerner) [inline desktop-winui/README.md:46]:
```suggestion
> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Microsoft Entra admin center, while `code formatting` indicates a value you enter into a text box in the Microsoft Entra admin center.

```

File: desktop-winui/README.md [hunk 11]
```diff
@@ -55,10 +55,10 @@ Finally, set the following values in _MainWindow.xaml.cs_.
 
 
 ```csharp
-// 'Tenant ID' of your Azure AD instance - this value is a GUID
+// 'Tenant ID' of your Entra AD instance - this value is a GUID
```
Comment (cilwerner) [inline desktop-winui/README.md:58]:
```suggestion
// 'Tenant ID' of your Microsoft Entra instance - this value is a GUID

```

File: console-cli/README.md
Comment (cilwerner) [inline console-cli/README.md:1]: Make changes in the Program.cs and you can then directly copy them over to this, as its a snippet

File: console-cli/Program.cs (hunk 4 above)
Comment (henrymbuguakiarie) [inline console-cli/Program.cs:8]:
```suggestion
    TenantId = "Enter_the_Tenant_ID_Here",

```

File: console-cli/Program.cs (hunk 5 above)
Comment (henrymbuguakiarie) [inline console-cli/Program.cs:11]:
```suggestion
    ClientId = "Enter_the_Application_Id_Here"

```

File: console-daemon/Program.cs
Comment (cilwerner) [inline console-daemon/Program.cs:1]: Bulk change `ClientID = ""` to add the Enter_Application_ID_Here (can't remember top of mind.)

File: console-cli/README.md [hunk 12]
```diff
@@ -61,15 +61,15 @@ Use these settings in your app registration.
...
 
-> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Microsoft Admin center, while `code formatting` indicates a value you enter into a text box in the Admin center.
 
 ### 2. Update the _Program.cs_ file with app registration values
 
 ```csharp
-// 'Directory (tenant) ID' of app registration in the Azure portal - this value is a GUID
+// 'Directory (tenant) ID' of app registration in the Admin center - this value is a GUID
 TenantId = "",
```
Comment (henrymbuguakiarie) [inline console-cli/README.md:70]:
```suggestion
TenantId = "Enter_the_Tenant_ID_Here",
```

File: desktop-winforms/README.md
Comment (cilwerner) [inline desktop-winforms/README.md:1]: Microsoft Entra admin center

File: console-cli/README.md [hunk 13]
```diff
@@ -61,15 +61,15 @@ Use these settings in your app registration.
...
 
-// 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+// 'Application (client) ID' of app registration in Admin center - this value is a GUID
 ClientId = ""
```
Comment (henrymbuguakiarie) [inline console-cli/README.md:73]:
```suggestion
ClientId = "Enter_the_Application_Id_Here"
```

File: desktop-wpf/MainWindow.xaml.cs [hunk 14]
```diff
@@ -28,10 +28,10 @@ public MainWindow()
             _msalPublicClientApp = PublicClientApplicationBuilder
                 .CreateWithApplicationOptions(new PublicClientApplicationOptions
                 {
-                    // 'Tenant ID' of your Azure AD instance - this value is a GUID
+                    // 'Tenant ID' of your Entra ID instance - this value is a GUID
```
Comment (cilwerner) [inline desktop-wpf/MainWindow.xaml.cs:31]:
```suggestion
                    // 'Tenant ID' of your Microsoft Entra instance - this value is a GUID

```

File: console-daemon/Program.cs (hunk 6 above)
Comment (henrymbuguakiarie) [inline console-daemon/Program.cs:9]:
```suggestion
    Authority = "https://login.microsoftonline.com/Enter_the_Tenant_ID_Here",

```

File: desktop-wpf/README.md [hunk 15]
```diff
@@ -43,7 +43,7 @@ Use these settings in your app registration.
 | **Platform type**               | **Mobile and desktop applications**                                 | Required value for this sample                                                  |
 | **Redirect URIs**               | `http://localhost`                                                  | Required value for this sample                                                  |
 
-> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Microsoft Admin center, while `code formatting` indicates a value you enter into a text box in the Microsoft Admin center.
```
Comment (cilwerner) [inline desktop-wpf/README.md:46]:
```suggestion
> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Microsoft Entra admin center, while `code formatting` indicates a value you enter into a text box in the Microsoft Entra admin center.
```

File: desktop-wpf/README.md [hunk 16]
```diff
@@ -54,10 +54,10 @@ Next, open the _MsalExamples.csproj_ project in Visual Studio.
 Finally, set the following values in _MainWindow.xaml.cs_.
 
 ```csharp
-// 'Tenant ID' of your Azure AD instance - this value is a GUID
+// 'Tenant ID' of your Entra ID instance - this value is a GUID
```
Comment (cilwerner) [inline desktop-wpf/README.md:57]:
```suggestion
// 'Tenant ID' of your Microsoft Entra instance - this value is a GUID
```

File: console-daemon/Program.cs (hunk 7 above)
Comment (henrymbuguakiarie) [inline console-daemon/Program.cs:13]:
```suggestion
    ClientSecret = "Enter_the_Client_Secrect_Here",

```

File: console-daemon/Program.cs [hunk 17]
```diff
@@ -7,11 +7,11 @@
 var config = new {
     // Full directory URL, in the form of https://login.microsoftonline.com/<tenant_id>
     Authority = "",
-    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    // 'Application (client) ID' of app registration in Microsoft Admin center - this value is a GUID
     ClientId = "",
```
Comment (henrymbuguakiarie) [inline console-daemon/Program.cs:11]:
```suggestion
    ClientId = "Enter_the_Application_Id_Here",

```

File: console-daemon/Program.cs (hunk 8 above)
Comment (henrymbuguakiarie) [inline console-daemon/Program.cs:15]:
```suggestion
    ClientObjectId = "Enter_the_Object_ID_Here"

```

File: web-app-aspnet/README.md [hunk 18]
```diff
@@ -41,19 +41,19 @@ Use the following settings for your app registration:
...
 1. Open the _~/sign-in-webapp/WebApp.csrpoj_ in your code editor.
 1. Open the _appsettings.json_ file and modify the following code:
 
     ```json
-    "TenantId": "[Enter 'common', or 'organizations' or the Tenant ID (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
-    "ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
-    "ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
+    "TenantId": "[Enter 'common', or 'organizations' or the Tenant ID (Obtained from the Entra admin center. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+    "ClientId": "[Enter the Client Id (Application ID obtained from the admin center), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
+    "ClientSecret": "[Copy the client secret added to the app from the admin center]",
```
Comment (cilwerner) [inline web-app-aspnet/README.md:56]:
```suggestion
    "TenantId": "Enter_'common'_or_'organizations'_the_Tenant_ID_Here",
    "ClientId": "Enter_the_Application_Id_Here",
    "ClientSecret": "Enter_the_Client_Secret_Value_Here",
```
Ok here is an outlier... are there others like this?

**Review by cilwerner — CHANGES_REQUESTED** [review]
A lot needed here, but also there may be things that I can't see

**Pull request closed without merging**